x-pack/test/apm_api_integration/tests/service_overview/dependencies/index.ts

```diff
@@ -441,38 +444,33 @@ export default function ApiTest({ getService }: FtrProviderContext) {
         const impactValues = sortBy(
           response.body.serviceDependencies.map((item) => ({
             name: item.name,
-            impact: item.impact,
-            latency: item.latency.value,
-            throughput: item.throughput.value,
+            currentPeriodImpact: item.currentPeriodMetrics.impact,
+            previousPeriodImpact: item.previousPeriodMetrics?.impact,
           })),
           'name'
         );
 
         expectSnapshot(impactValues).toMatchInline(`
           Array [
             Object {
-              "impact": 1.36961744704522,
-              "latency": 2568.40816326531,
+              "currentPeriodImpact": 1.54893576051104,
               "name": "elasticsearch",
-              "throughput": 13.0666666666667,
+              "previousPeriodImpact": 1.19757368986118,
```

Comment:
Why did impact change from "1.36961744704522" to "1.54893576051104"?

Reply:
That's because the value `1.36961744704522` was based on the entire 30 minutes of data before adding the comparison range. With the comparison time range filtering the result, it is only 15 minutes for the current period and 15 minutes for the previous period. That's why it changed to `1.54893576051104` and now it also has  `"previousPeriodImpact": 1.19757368986118,`